### PR TITLE
note added in subtext of journal section

### PR DESCRIPTION
**Pull request by Shubham-Praj**
This resolves issue #1130 

I have added note as a subtext in My journal listing section.

The note  properties are as follows 
**font size : 1rem** and **opacity : 0.8**

If you want me to change font-size please let me know the font-size , according to me this was better UI wise, so I implemented it.

Old screenshot 
image

New screenshot 
image

### CODE REVIEW

**Comment by tijlleenders** [comment]
@Shubham-Praj Very nice!

**Comment by Shubham-Praj** [comment]
Thank you 😁

**Review by Tushar-4781 — COMMENTED** [review]
@Shubham-Praj nice work it seems like it loads the entire note we have to apply a limit on number of lines. @tijlleenders how many lines of not should be visible in collapsed state ? I say 1 line is enough otherwise it will become too bulky once we have multiple feelings

**Comment by tijlleenders** [comment]
@Tushar-4781 Yes I think we can do a max of one line and ellipse ("...") if it is longer.

File: src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx [hunk 1]
```diff
@@ -111,7 +111,11 @@ export const ShowFeelingTemplate: React.FC<ShowFeelingTemplateProps> = ({
               >
                 <div className="feelingOfDay-name">
                   {feelingsEmojis[feelingsListObject[feelingId].category]}&nbsp;
-                  <span>{t(feelingsListObject[feelingId].content)}</span>
+                  <div>
```
Comment (Tushar-4781) [inline src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx:114]: @Shubham-Praj please pull latest changes again to resolve the merge conflicts.

in this div create a class and put this css ```
{
display: -webkit-box;
-webkit-line-clamp: 2;
-webkit-box-orient: vertical;
overflow: hidden; }
```

File: src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx [hunk 2]
```diff
@@ -111,7 +111,11 @@ export const ShowFeelingTemplate: React.FC<ShowFeelingTemplateProps> = ({
...
+                    <span>{t(feelingsListObject[feelingId].content)}</span>
+                    <br />
+                    <span style={{ fontSize: "1rem", opacity: "0.8" }}>{t(feelingsListObject[feelingId].note)}</span>
```
Comment (Tushar-4781) [inline src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx:117]: and change this css to -> ` {
    font-size: 0.875rem;
    opacity: 0.8;
    word-break: break-word;
 } `

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]
@Shubham-Praj please pull latest changes again to resolve the merge conflicts.
After doing below changes, the note won't go more than 1 line as well. The output should be like this
image

File: src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx (hunk 2 above)
Comment (Shubham-Praj) [reply src/pages/ShowFeelingsPage/ShowFeelingTemplate.tsx:117]: Got it

**Pull request closed without merging**